src/test/java/net/snowflake/client/core/SFTrustManagerIT.java

```diff
@@ -182,7 +182,7 @@ private static void accessHost(String host, HttpClient client) throws IOExceptio
     assertThat(
         String.format("response code for %s", host),
         statusCode,
-        anyOf(equalTo(200), equalTo(403), equalTo(400)));
+        anyOf(equalTo(200), equalTo(403), equalTo(400), equalTo(513)));
```

Comment:
Please also add 404. This change will come as soon as we start moving traffic to envoy.

Reply:
Added 404